src/test/java/marquez/client/models/ModelGenerator.java

```diff
@@ -29,6 +29,12 @@ private ModelGenerator() {}
 
   private static final Random RANDOM = new Random();
 
+  public static JobType newJobType() {
+    JobType[] jobTypes = JobType.values();
```

Comment:
_Minor:_ Another way to simplify randomizing a job type is:

```
JobType.values()[newIdWithBound(JobType.values().length - 1)];
```

You need to define this sweet helper method:

```java
protected static Integer newIdWithBound(final int bound) {
   return RANDOM.nextInt(bound);
}
```